bundle/Command/ImportTagsCommand.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Netgen\TagsBundle\Command;
+
+use Netgen\TagsBundle\API\Repository\Values\Tags\TagCreateStruct;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImportTagsCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('netgen:tags:import')
+            ->setDescription("Creates new tags based on an already existing CSV file. The file should be uploaded to the root of the installation.")
+            ->addOption(
+                'filename',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'The CSV file to be imported (NOTE: input the relative path from the root of the installation)'
+            )
+            ->addOption(
+                'parent-tag-id',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The ID of the tag underneath which to add the tags in the CSV file'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /** @var \eZ\Publish\API\Repository\Repository $repository
+         */
+        $repository = $this->getContainer()->get('ezpublish.api.repository');
+
+        $currentUserReference = $repository->getPermissionResolver()->getCurrentUserReference();
+
+        $adminUserReference = new UserReference(14);
+
+        // We need to set the current user to admin until the import is ready since the anonymous user which is logged
+        // in by default does not have a permission to add a tag.
+        $repository->getPermissionResolver()->setCurrentUserReference($adminUserReference);
+
+        $filename = $input->getOption('filename');
+
+        if (empty($filename)) {
+            throw new \RuntimeException('Parameter --filename is required');
+        }
+
+        $parentTagId = $input->getOption('parent-tag-id');
+
+        if (empty($parentTagId)) {
+            $parentTagId = 0;
+        }
+
+        $fileFullPath = $this->getContainer()->getParameter('kernel.root_dir') . $filename;
```

Comment:
There's no need for this. `file_get_contents` and `file_exists` will read the relative path to file based on the current working directory, while keeping compatibility with full paths automatically. So just using the plain option value will work as well.